corehq/privileges.py

```diff
@@ -99,4 +99,13 @@ def get_name_from_privilege(cls, privilege):
             REPORT_BUILDER: _('User Configurable Report Builder'),
             TEMPLATED_INTENTS: _('Templated Intents'),
             CUSTOM_INTENTS: _('Custom Intents'),
-        }.get(privilege, privilege)
+            DATA_CLEANUP: _('Data Management'),
+        }
+        if privilege in titles:
+            return titles[privilege]
+        else:
+            from corehq.util.soft_assert import soft_assert
+            _assert = soft_assert(notify_admins=True)
+            _assert(False, '{} does not have a title defined, users are '
+                           'seeing the slug itself.'.format(privilege))
```

Comment:
Looks like it's used in only two places - the `SubscriptionUpgradeRequiredView`, which I expect is pretty low-traffic, and [here](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/accounting/decorators.py#L35), which gets called for domains on trial subscriptions for views with that decorator - so probably a fair bit, but less than once per request.